src/utils/seedDB.js

```diff
@@ -1,7 +1,9 @@
-const createUsersMockData = require('./mock_data/createUsersMockData');
```

Comment:
The renaming of the file would have been done automatically when I merged my code in. There might be some errors once we merge both of our codes (associated with the renamed file), but no worries, I'll fix it if the issue arises.